src/main/steps/common/task-list/components/notification-banner/utils.ts

```diff
@@ -116,3 +122,11 @@ export const isApplicantLIPServingRespondent = (caseData: Partial<CaseWithId>):
 export const isPrimaryApplicant = (caseData: Partial<CaseWithId>, userDetails: UserDetails): boolean => {
   return caseData.applicants?.[0].value.user.idamId === userDetails.id;
 };
+
+export const isPartyServed = (
+  caseData: Partial<CaseWithId>,
+  userDetails: UserDetails,
+  partyType: PartyType
+): boolean => {
+  return getCurrentPartyId(caseData, userDetails, partyType) === caseData.citizenApplicationPacks?.[0].partyId;
```

Comment:
u need to check if citizenApplicationPacks has any item before you call getCurrentPartyId